### PR TITLE
Metricbeat - apache: Move regexp.MustCompile to 'var' for performance improvement

### PR DESCRIPTION
**Pull request by radoondas**
Hope this PR works.

### CODE REVIEW

**Comment by ruflin** [comment]
jenkins, test it

File: metricbeat/module/apache/status/data.go [hunk 1]
```diff
@@ -11,239 +11,120 @@ import (
...
-			totalW = strings.Count(scr[1], "W")
-			totalK = strings.Count(scr[1], "K")
-			totalD = strings.Count(scr[1], "D")
-			totalC = strings.Count(scr[1], "C")
-			totalL = strings.Count(scr[1], "L")
-			totalG = strings.Count(scr[1], "G")
-			totalI = strings.Count(scr[1], "I")
-			totalDot = strings.Count(scr[1], ".")
-			totalAll = totalUnderscore + totalS + totalR + totalW + totalK + totalD + totalC + totalL + totalG + totalI + totalDot
+			logp.Warn("Unexpected line in apache server-status output: %s", scanner.Text())
```
Comment (ruflin) [inline metricbeat/module/apache/status/data.go]: Hm, this is still Warn?

Comment (radoondas) [inline metricbeat/module/apache/status/data.go]: Ouch, yes :( fix in 2 minutes.

**Comment by radoondas** [comment]
@ruflin message fixed.

**Comment by ruflin** [comment]
jenkins, test it

**Pull request merged**